### PR TITLE
Redesign vote writing screen

### PR DESCRIPTION
**Pull request by harrygr**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -41,8 +41,12 @@ injectGlobal`
     background-repeat: no-repeat;
     background-position: center center;
     background-size: 38rem;
+    font-size: 16px;
 }
   }
+  h1,h2,h3,h4,h5 {
```
Comment (Jarrydcrawford) [inline src/App.tsx]: Poor h6, forgotten about #thisissosad

File: src/components/MatchInfo.tsx [hunk 2]
```diff
@@ -2,33 +2,62 @@ import { format } from 'date-fns';
...
+
+const VersesContainer = styled('div')`
+  display: grid;
+  grid-template-columns: 1fr auto 1fr;
+  grid-column-gap: ${spacing.goat};
+  font-size: ${typeSize.llama};
+  > :first-child {
+    text-align: right;
+  }
+`;
```
Comment (Jarrydcrawford) [inline src/components/MatchInfo.tsx]: I'm okay for grid use if we're not supporting IE{X}

File: src/components/MatchInfo.tsx [hunk 3]
```diff
@@ -2,33 +2,62 @@ import { format } from 'date-fns';
...
-        {format(match.playedAt, 'DD MMM YYYY')}
+        {format(match.playedAt, 'Qo MMM YYYY')}
       </MatchDate>
-    </InfoContainer>
+      <Verses team1={match.team.name} team2={match.opponent} />
+    </Vspace>
+  );
+};
+
+interface VersesProps {
```
Comment (Jarrydcrawford) [inline src/components/MatchInfo.tsx]: Versus 🤔

File: src/components/MatchInfo.tsx [hunk 4]
```diff
@@ -2,33 +2,62 @@ import { format } from 'date-fns';
...
+  }
+`;
+
+const VersesSeparator = styled('div')`
+  font-size: ${typeSize.rhino};
+  text-align: center;
+  line-height: 1;
+`;
+
+const Verses = ({ team1, team2 }: VersesProps) => {
```
Comment (Jarrydcrawford) [inline src/components/MatchInfo.tsx]: Something more semantic like `homeTeam` and `awayTeam` or something like that. Can't think of anything else right now 💤

File: src/components/TextArea.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+import { baseInputStyles, LabelContent } from './Form';
+import { InputError } from './InputError';
+import { Vspace } from './Vspace';
+
+interface Props
+  extends React.DetailedHTMLProps<
+      React.TextareaHTMLAttributes<HTMLTextAreaElement>,
+      HTMLTextAreaElement
+    > {
```
Comment (Jarrydcrawford) [inline src/components/TextArea.tsx]: That's a fat interface generic chain to get the textarea props 😂

File: src/components/TextArea.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  display: block;
+  width: 100%;
+`;
+
+export const TextArea = ({
+  label,
+  error,
+  className,
+  ref: _ref,
+  ...inputprops
```
Comment (Jarrydcrawford) [inline src/components/TextArea.tsx]: Did you need to include ref and the rest props? 🤷‍♂️

File: src/pages/CreateVote/SelectableCard.tsx [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+const SelectableCardContainer = styled('li')`
+  :not(:first-child) {
+    margin-left: ${spacing.goat};
+  }
+  > input {
+    display: none;
+  }
+  display: block;
```
Comment (Jarrydcrawford) [inline src/pages/CreateVote/SelectableCard.tsx]: This should be above the child selectors at the top of the styled tag template literal.

File: src/components/TextArea.tsx (hunk 6 above)
Comment (harrygr) [reply src/components/TextArea.tsx]: I had to remove the ref from the rest in order to satisfy the compiler. `inputprops` is provided to the `StyledTextArea` and `ref` is not accepted by a styled (emotion) component.

File: src/App.tsx (hunk 1 above)
Comment (harrygr) [reply src/App.tsx]: I doubt we'll ever use an h6 but I'll add it all the same

File: src/components/MatchInfo.tsx (hunk 2 above)
Comment (harrygr) [reply src/components/MatchInfo.tsx]: yeah IE6 can do one

File: src/components/MatchInfo.tsx (hunk 3 above)
Comment (harrygr) [reply src/components/MatchInfo.tsx]: Could most likely be better named, and moved closer to the component

File: src/components/TextArea.tsx (hunk 5 above)
Comment (harrygr) [reply src/components/TextArea.tsx]: Yeah, but it gives the best types for the component and avoids declaring a load of our own types

**Pull request merged**